e2e/test/scenarios/embedding/sdk-iframe-embedding/sdk-iframe-embedding.cy.spec.ts

```diff
@@ -144,7 +144,7 @@ describe("scenarios > embedding > sdk iframe embedding", () => {
       });
   });
 
-  it("does not allow updating instanceUrl via embed.updateSettings", () => {
+  it("does not allow changing the value of instanceUrl via embed.updateSettings", () => {
```

Comment:
This is for when `updateSettings` tries to set a different value of instanceUrl than the one we have initially. There's another test case below for the opposite case.